apps/web/src/server/appeals/appeal-details/appeal-details.router.js

```diff
@@ -5,4 +5,26 @@ const router = createRouter();
 
 router.route('/:appealId').get(controller.viewAppealDetails);
 
+// TODO: BOAT-103, BOAT-104 (router and controller for setting start date)
+import logger from '../../lib/logger.js';
+import * as appealDetailsService from './appeal-details.service.js';
+
+router.route('/:appealId/enter-start-date').get(async (request, response) => {
+	const appealDetails = await appealDetailsService
+		.getAppealDetailsFromId(request.params.appealId)
+		.catch((error) => logger.error(error));
+
+	if (appealDetails) {
+		const appealReferenceFragments = appealDetails?.appealReference.split('/');
```

Comment:
Perhaps it would be better to use the `appealShortReference` nunjucks filter (apps/web/src/server/lib/nunjucks-filters/appeals.js) here for consistency.

Reply:
Ah good shout, I've updated

Reply:
:+1: